src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/src/ExceptionAssertions.cs

```diff
@@ -0,0 +1,272 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    // TODO: eventually want: public partial class Assert : Xunit.Assert
+    public static class ExceptionAssert
```

Comment:
We already have such a file: https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/AssertExtensions.cs